src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -148,8 +149,8 @@ export class ServiceSelectionScene extends SceneObjectBase<ServiceSelectionScene
     }
 
     try {
-      const serviceSearch = service ? `(?i).*${service}.*` : '.+';
-      const volumeResponse = await ds.getResource!(
+      const serviceSearch = service ? `(?i).*${escapeRegExp(service)}.*` : '.+';
+      const volumeResponse = await ds.getResource(
         'index/volume',
         {
           query: `{${SERVICE_NAME}=~"${serviceSearch}"}`,
```

Comment:
```suggestion
          query: `{${SERVICE_NAME}=~\`${serviceSearch}\`}`,
```

should work with regex escaping

Reply:
Perfect! 

![imagen](https://github.com/grafana/explore-logs/assets/1069378/ff8db0a9-7926-4eb3-9ecb-6eb45851cdb2)